sources/Application/Utils/HelpLegend.h

```diff
@@ -0,0 +1,98 @@
+#include <cstdio>
+#include <cstring>
+
+static char** getHelpLegend(FourCC command) {
+  static char* result[2];
+  result[1] = (char*)("bb at speed aa");
+  switch (command)
+  {
+    case 1:
```

Comment:
Thanks so much for persevering with this! This is all good now, but could I just ask for 1 more change: could you use the defines for the command indexes instead of the number values directly, so `case I_CMD_ARPG:` instead of just `case 3:` 🙏🏻

Reply:
Noooo i staged parts of my test class >___<;
Note to self: don't commit at 05:30 in the morning.